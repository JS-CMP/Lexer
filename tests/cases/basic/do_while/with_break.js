do {
    console.log("Loop running")
    break
} while (true)
