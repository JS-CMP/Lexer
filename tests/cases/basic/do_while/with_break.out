#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"

int main() {do {console["log"](JS::Any("Loop running"));break;}while (JS::Any(true));}
